openpype/modules/slack/plugins/publish/integrate_slack_api.py

```diff
@@ -60,8 +61,10 @@ def process(self, instance):
                 else:
                     client = SlackPython3Operations(token, self.log)
 
-                users, groups = client.get_users_and_groups()
-                message = self._translate_users(message, users, groups)
+                if "@" in message:
+                    if users is None:
+                        users, groups = client.get_users_and_groups()
```

Comment:
What's the speed of `client.get_users_and_groups()`? Is it **very slow?**

Since this is an InstancePlugin would it be maybe worth caching the result in `context.data` on the first run and any consecutive instances would re-use the value.

For e.g. 10 instances that'd bring it back from 10 calls to a single call.